src/elements.cpp

```diff
@@ -64,7 +64,7 @@ G1Element G1Element::FromBytes(const Bytes& bytes)
 
 G1Element G1Element::FromByteVector(const std::vector<uint8_t>& bytevec)
```

Comment:
the `FromByteVector()` functions don't really serve a purpose with this patch anymore, since the `Bytes` overload can be used for `std::vector` as well. Since it's part of the public API though, I'm hesitant to remove it.